pos_payment_method_cashdro/models/pos_payment_method.py

```diff
@@ -13,10 +13,11 @@ def _get_payment_terminal_selection(self):
         string="Cashdro Terminal Host Name or IP address",
         help="It must be reachable by the PoS in the store",
     )
-    cashdro_user = fields.Char(string="Cashdro User")
-    cashdro_password = fields.Char(string="Cashdro Password")
+    cashdro_user = fields.Char()
+    cashdro_password = fields.Char()
 
-    def _onchange_is_cash_count(self):
+    def _onchange_journal_id(self):
+        res = super()._onchange_journal_id()
         if self.use_payment_terminal == "cashdro":
-            return
-        return super()._onchange_is_cash_count()
+            self.use_payment_terminal = True
```

Comment:
Don't get the intention of this changing the value from `cashdro` to `True`, and there's no docstring.

Reply:
It wasn't right indeed. Fixed and added docstring